src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -148,17 +166,17 @@ const Actions: React.FC<Props> = ({
                     anchorEl={actionAnchorElement}
                 >
                     <MenuList id="action-menu">
-                        {Object.entries(availableActions).map(([key]) => {
+                        {availableActionsForUserRole.map(({ actionName }) => {
                             return (
                                 <MenuItem
-                                    key={key}
+                                    key={actionName}
                                     onClick={onMenuItemClick(
-                                        key as ActionName,
+                                        actionName as ActionName,
```

Comment:
Why do we need the type assertion "as ActionName", shouldn't typescript know it is an ActionName?

Edit - was it a relic of the old key value map -- another reason for this refactor :)